app/util/util.ts

```diff
@@ -9,6 +9,7 @@ const sendTxn = async (bitcoinAddress: Address, sendAmount: number, opReturnInpu
 		// fetch utxos
 		const utxos: UTXO[] = await fetchUTXOs(bitcoinAddress.address);
 		if (!utxos?.length) {
+			// TODO: Also add better notification handling. Task -> https://github.com/gonative-cc/byield/issues/21
 			throw new Error("utxos not found.");
```

Comment:
let's add log. 
Exception is bad because it breaks the app (we can keep it until proper user message will be added)

Reply:
Yes. I have added the console.error. Thanks.